ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -146,6 +146,13 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    /// Indicates if the sheet should always fill the available width.
```

Comment:
Should document the default value since most would assume the default value of a Bool is false